lib/components/form.js

```diff
@@ -42,21 +42,26 @@ const Form = React.createClass({
       isAgreed: !this.state.isAgreed
     })
   },
-  componentDidMount() {
-    let { account, repository } = this.props.params
-    let json = `${endpoints.rawgit}${account}/${repository}/master/issue.json`
-    this.setState({
-      account: account,
-      repository: repository
-    })
+  fetchData(account, repository) {
```

Comment:
It's necessary to check. However, If account or repository was inputted, Where will the user go?